app/content/react/library-reference/api.md

```diff
@@ -64,16 +64,20 @@ React hook that allows a more powerful use of CARTO deck.gl layers, creating a s
 {{% tableWrapper tab="true" %}}
 | Param              | Type                | Description              |
 | ------------------ | ------------------- | ------------------------ |
-| props              | <code>Object</code> | Object properties        |
-| props.source       | <code>Object</code> | Data source              |
-| props.source.id    | <code>string</code> | Unique source ID         |
+| props              | <code>Object</code> | { source, [layerConfig], [uniqueIdProperty], [viewportFeatures], [viewportFeaturesDebounceTimeout]}  |
+| props.source       | <code>Object</code> | { id, type, connection, data, [credentials] }              |
+| props.source.id    | <code>string</code> | Unique source ID.         |
 | props.source.type  | <code>string</code> | Source type. Check available types [here](/deck-gl/reference#type-string)  |
 | props.source.connection  | <code>string</code> | Connection name. Required only for CARTO 3.  |
-| props.source.data  | <code>string</code> |  Table name, tileset name or SQL query                                           |
-| props.source.credentials | <code>string</code> |  Credentials for accessing the source (check the parameters [here](/deck-gl/reference#setdefaultcredentials))                                           |
-| [props.uniqueIdProperty] | <code>string</code> | (optional) Name of the column for identity. To be used internally when getting viewportFeatures (used by widget computations) |
+| props.source.data  | <code>string</code> |  Table name, tileset name or SQL query. |
+| [props.source.credentials] | <code>string</code> |  Credentials for accessing the source (check the parameters [here](/deck-gl/reference#setdefaultcredentials)).                                           |
+| [props.layerConfig]       | <code>Object</code> | { id, opacity, visible }        |
+| [props.layerConfig.id]    | <code>string</code> | Unique layer ID.                |
+| [props.layerConfig.opacity]  | <code>number</code> | Initial layer opacity.       |
+| [props.layerConfig.visible]  | <code>boolean</code> | Indicates whether the layer is visible by default or not. |
```

Comment:
I'd indicate the default values if layerConfig isn't provided. opacity is 1 and visible is true.

Reply:
Default values added.